lib/src/screens/memory/memory_screen.dart

```diff
@@ -138,15 +138,17 @@ class MemoryScreen extends StatelessWidget {
                   ),
                 ),
                 child: ModelView(
-                  state.memory.modelViewEvent ?? AppEvent(),
+                  state.memory.modelViewEvent?.modelData ?? '{}',
```

Comment:
ModelView should use AppEvent. In future we need to show model name, timestamp and any other data, AppEvent comes in hand.